build/yamls/base/kustomization.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: flow-visibility
```

Comment:
shall we start using 'theia' as a namespace name (or a prefix?)

Reply:
Can I ask about the motivation for including 'theia' in the namespace name? It seems to me we are not including 'antrea' in any namespace name in antrea repo.

Reply:
Indeed it was just a question, and your motivation is good enough!